command/resource/client/grpc-client_test.go

```diff
@@ -49,3 +51,57 @@ func TestResourceRead(t *testing.T) {
 		prototest.AssertDeepEqual(t, writeRsp.Resource, readRsp.Resource)
 	})
 }
+
+func TestResourceReadInTLS(t *testing.T) {
```

Comment:
Could we also test with:

* tls verification disabled (no ca certs provided)
* no client certs

Reply:
Yup, added